wfe2/wfe_test.go

```diff
@@ -1603,31 +1602,54 @@ func TestEmptyAccount(t *testing.T) {
 	_, ok := key.(*rsa.PrivateKey)
 	test.Assert(t, ok, "Couldn't load RSA key")
 
-	payload := `{}`
-	path := "1"
-	signedURL := "http://localhost/1"
-	_, _, body := signer.byKeyID(1, key, signedURL, payload)
-	request := makePostRequestWithPath(path, body)
-
-	// Send an account update with the trivial body
-	wfe.Account(
-		ctx,
-		newRequestEvent(),
-		responseWriter,
-		request)
+	testCases := []struct {
+		Name      string
+		payload   string
+		path      string
+		signedURL string
+	}{
+		{
+			Name:      "POST empty string to acct",
+			payload:   "",
+			path:      "1",
+			signedURL: "http://localhost/1",
```

Comment:
These are the same in all test cases, so they don't necessarily need to be here.

Totally optional improvement: you could include some "empty-like" inputs, such as `"\"\""` and "[]", which should result in errors.